src/Elm/Print.hs

```diff
@@ -421,26 +428,211 @@ typeRefEncoder (RefPrim elmPrim) = case elmPrim of
     ElmList l     -> "E.list" <+> typeRefEncoder l
 typeRefEncoder (RefCustom TypeName{..}) = "encode" <> pretty unTypeName
 
-encodeMaybe :: Doc ann
-encodeMaybe = vsep
+encodeMaybe :: Text
+encodeMaybe = T.unlines
```

Comment:
Nice idea! Since it's just a raw `Text`, don't need to use `showDoc` function. You can even remove it from exports 👍